package/cpp/api/JsiSkTypeFaceFontProvider.h

```diff
@@ -46,7 +47,8 @@ class JsiSkTypefaceFontProvider
   JSI_HOST_FUNCTION(matchFamilyStyle) {
     auto name = arguments[0].asString(runtime).utf8(runtime);
     auto fontStyle = JsiSkFontStyle::fromValue(runtime, arguments[1]);
-    auto typeface = getObject()->matchFamilyStyle(name.c_str(), *fontStyle);
+    sk_sp<SkFontStyleSet> set(getObject()->onMatchFamily(name.c_str()));
+    sk_sp<SkTypeface> typeface(set->matchStyle(*fontStyle));
```

Comment:
onMatchFamily can return NULL if no font family matches, and so this will crash. @wcandillon